schema/library.json

```diff
@@ -532,5 +532,291 @@
         ],
         "description_verification_type": "rule-based",
         "assertions_type": "pass"
-    }
+    },
+    "G36_SupplyAirTemperatureSetpoint": {
```

Comment:
@leijerry888 FYI, Jeremy and I discussed that the names should match the class name. Also, as of ```G36_SimultaneousHeatingCooling```, there are 2 more spaces so, this will be fixed soon.

Reply:
Fixed in the latest commit!